project_agile/models/mixins.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+import logging
+from odoo import models, api
+
+_logger = logging.getLogger(__name__)
+
+
+class IdSearchMixin(models.AbstractModel):
```

Comment:
IMO should be in separate module

Reply:
I agree. I'll move into separate module but for which repository should I create pull request?
Do we leave it here in this repo or maybe you have a suggestion?